### PR TITLE
Add FAQs page to docs

### LINKED ISSUE #162: conflicting values for variable 'lat_bnds'
### What versions of software are you using?

-   Package Version: Main branch / Release 0.1.0

### What are the steps to reproduce this issue?
```
import xcdat

p = '/p/css03/scratch/cmip6/CMIP/NCC/NorCPM1/historical/r10i1p1f1/Amon/tas/gn/v20190914/*nc'
ds = xcdat.open_mfdataset(p)
```

> MergeError: conflicting values for variable 'lat_bnds' on objects to be combined. You can skip this check by specifying compat='override'.

### What happens? Any logs, error output, etc?

I think this is happening because the `lat_bnd` values have slight (e.g., 10<sup>-13</sup>) differences in the different netCDF files.

### Any other comments?
Opening with xarray works fine (as long as you do not set `data_vars="minimal"`), though the `lat_bnd` is larger than expected:

```
import xarray as xr
ds = xr.open_mfdataset(p)
print(ds.lat_bnds.shape)
```
> (2160, 142, 2)

It appears that the bounds differ depending on timestep:
```
np.array(ds.lat_bnds[0] - ds.lat_bnds[-1])
```

> array([[-9.00000000e+01, -8.90526316e+01],
>           [-8.90526316e+01, -8.71578947e+01],
>        [-8.71578947e+01, -8.52631579e+01],
> ...
>        [            nan,             nan],
>        [ 8.90526316e+01,  9.00000000e+01],
>        [            nan,             nan]])

I'm not totally sure why there are `NaN` values. They don't appear when I use `ncdump -v lat_bnds /p/css03/scratch/cmip6/CMIP/NCC/NorCPM1/historical/r10i1p1f1/Amon/tas/gn/v20190914/tas_Amon_NorCPM1_historical_r10i1p1f1_gn_185001-201412.nc`, but they do appear after loading the dataset with xarray. 

I do not know what to do with this problem, though CDAT seems to be able to load the bounds with no problem.

### PR DESCRIPTION
**Pull request by tomvothecoder**
## Description

This PR adds a page to the documentation for FAQs. It also adds a section to the FAQs for handling #162.

- Closes #162 

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

If applicable:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass with my changes (locally and CI/CD build)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have noted that this is a breaking change for a major release (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: docs/faqs.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Let's say you try opening two files using ``xcdat.open_mfdataset()`` and the aforementioned ``MergeError`` appears for the ``lat_bnds`` data var.
+You perform a floating point comparison between both files for ``lat_bnds`` and find a very small floating point difference at specific coordinates.
+
+To workaround this data quality issue and proceed with opening the files, pass these keyword arguments:
+
+1. ``compat="override"``: skip comparing and pick variable from first dataset
+2. ``join="override"``:  if indexes are of same size, rewrite indexes to be those of the first object with that dimension. Indexes for the same dimension must have the same size in all objects.
+
+   - ``join`` is set to `"outer_join"`, which might not be desired.
```
Comment (tomvothecoder) [inline docs/faqs.rst:18]:
```suggestion
   - ``join`` is set to `"outer_join"` by default, which might not be desired.
```

File: docs/faqs.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
+==========================
+Frequently Asked Questions
+==========================
+
+How do I open files that have conflicting values for a data variable(s)?
+------------------------------------------------------------------------
+In xarray, the default setting for checking compatibility across files is ``compat='no_conflicts'``.
+If conflicting values exist for a data variable between the files, xarray raises ``MergeError: conflicting values for variable <DATA VAR NAME> on objects to be combined. You can skip this check by specifying compat="override".``
```
Comment (tomvothecoder) [inline docs/faqs.rst:8]:
```suggestion
If conflicting values for a data variable between the files exists, xarray raises ``MergeError: conflicting values for variable <DATA VAR NAME> on objects to be combined. You can skip this check by specifying compat="override".``
```

File: docs/faqs.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+To workaround this data quality issue and proceed with opening the files, pass these keyword arguments:
+
+1. ``compat="override"``: skip comparing and pick variable from first dataset
+2. ``join="override"``:  if indexes are of same size, rewrite indexes to be those of the first object with that dimension. Indexes for the same dimension must have the same size in all objects.
+
+   - ``join`` is set to `"outer_join"` by default, which might not be desired.
+
+``xcdat.open_mfdataset('path/to/files/*.nc', compat="override", join="override")``
+
+More information here: https://xarray.pydata.org/en/stable/generated/xarray.open_mfdataset.html#xarray-open-mfdataset
```
Comment (tomvothecoder) [inline docs/faqs.rst:22]: @pochedls Just an FYI

Comment (pochedls) [reply docs/faqs.rst:22]: This is great - thanks Tom!

**Pull request merged**